lighthouse-viewer/app/src/main.js

```diff
@@ -43,3 +43,10 @@ Promise.all(loadPolyfillPromises).then(_ => {
   // eslint-disable-next-line no-new
   new LighthouseViewerReport();
 });
+
+if ('serviceWorker' in navigator) {
+  navigator.serviceWorker.register('sw.js').catch(e => {
+    // eslint-disable-next-line no-console
```

Comment:
since this file is only for running in the browser, want to just do `/* eslint-disable no-console */` at the top of the file? Or is it better to keep it around to cut down on `console` calls left over from development?

(the rule is a bit annoying for debugging in viewer and in the extension since both have the `gulp lint` task as a required prerequisite for `gulp build`, but it's not the end of the world)